spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 ENV["RAILS_ENV"] ||= 'test'
 
+require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
+
```

Comment:
Under parallel tests we have https://github.com/codeclimate/ruby-test-reporter#installation

require 'simplecov'
require 'codeclimate-test-reporter'
SimpleCov.add_filter 'vendor'
SimpleCov.formatters = []
SimpleCov.start CodeClimate::TestReporter.configuration.profile

Then after all your tests run, in a rake task or as a build step do:

require 'simplecov'
require 'codeclimate-test-reporter'
CodeClimate::TestReporter::Formatter.new.format(SimpleCov.result)